pkg/envoy/bootstrap/config_test.go

```diff
@@ -382,5 +383,352 @@ static_resources:
         '@type': type.googleapis.com/envoy.extensions.filters.listener.tls_inspector.v3.TlsInspector
     name: startup_listener
 `
-	assert.Equal(expectedYAML, string(actualYAML))
+
```

Comment:
I'd recommend simply sorting the probes when we convert to yaml so that we get a deterministic order.